spec/web_helpers_spec.rb

```diff
@@ -0,0 +1,23 @@
+def full_strike_frame
```

Comment:
The file name is probably not best, as this is not "web" based.

Reply:
The content of the file is good however, but be careful, sometimes extracting test helpers can make tests less readable, so it's a balance.